### PR TITLE
Add $wp_config_path param for  get_wp_config_code

### PR DESCRIPTION
**Pull request by pmbaldha**
Issue: https://github.com/wp-cli/ideas/issues/138

### CODE REVIEW

**Comment by schlessera** [comment]
This should have a test in https://github.com/wp-cli/wp-cli/blob/master/features/wp-config.feature to make sure we don't introduce a regression later on.

**Comment by pmhost9** [comment]
@schlessera I am developing behat tests for it.

**Comment by pmbaldha** [comment]
@schlessera I am developing behat tests for it.

**Comment by pmhost9** [comment]
@schlessera  I have added all behat tests for --config-file param in the https://github.com/wp-cli/config-command/pull/104/files

I am not understanding which tests I should write for this PR.

**Comment by pmbaldha** [comment]
@schlessera I have added all behat tests for --config-file param in the https://github.com/wp-cli/config-command/pull/104/files

I am not understanding which tests I should write for this PR.

**Comment by schlessera** [comment]
@pmbaldha My bad, I don't think we need a test here. This is not externally accessible, and the Behat test from the other package already tests it in terms of safeguarding it against a regression.

File: php/WP_CLI/Runner.php [hunk 1]
```diff
@@ -595,10 +595,14 @@ public function is_command_disabled( $command ) {
 	/**
 	 * Returns wp-config.php code, skipping the loading of wp-settings.php
 	 *
+	 * @param string $wp_config_file_path config file path
```
Comment (schlessera) [inline php/WP_CLI/Runner.php:598]:
```suggestion
	 * @param string $wp_config_file_path Optional. Config file path. If left empty, it tries to
	 *                                    locate the wp-config.php file automatically.

File: php/WP_CLI/Runner.php [hunk 2]
```diff
@@ -595,10 +595,14 @@ public function is_command_disabled( $command ) {
...
+	 *
 	 * @return string
 	 */
-	public function get_wp_config_code() {
-		$wp_config_path = Utils\locate_wp_config();
+	public function get_wp_config_code( $wp_config_path = '' ) {
+		if ( '' === $wp_config_path ) {
```
Comment (schlessera) [inline php/WP_CLI/Runner.php:603]: Using `empty()` here will be safer and also deal with `null`.
```suggestion
		if ( empty( $wp_config_path ) ) {
```

File: php/WP_CLI/Runner.php [hunk 3]
```diff
@@ -595,10 +595,15 @@ public function is_command_disabled( $command ) {
 	/**
 	 * Returns wp-config.php code, skipping the loading of wp-settings.php
 	 *
+	 * @param string $wp_config_file_path config file path Optional. Config file path. If left empty, it tries to
```
Comment (schlessera) [inline php/WP_CLI/Runner.php:598]:
```suggestion
	 * @param string $wp_config_file_path Optional. Config file path. If left empty, it tries to
```

**Comment by schlessera** [comment]
Note: GitHub allows you to directly commit suggestions like the one I made above via the `Commit suggestion` button below the comment. That is generally less error-prone.

**Pull request merged**